src/SmartLeds.h

```diff
@@ -30,7 +30,7 @@
 
 #include <memory>
 #include <cassert>
-
+_PTR	 _EXFUN(memset,(_PTR, int, size_t));
```

Comment:
Co má prosím dělat tento řádek?

Reply:
Nebyl jsem si jist, co dělá memset(případně kde mimo arduino je definovaný), tak jsem ho zkopíroval z té verze, kterou používá Arduino.

Reply:
Nerozumím. `memset` je funkce ze standardní Cčkové knihovny. Dívám se, že mi chybí `#include <cstring>` a proto se ti to nedařilo zkompilovat. Zruš prosím tuto hrůzostrašnost a přidej správný include.

Reply:
memeset co pužíváte v SmartLeds.h 310, je přidaný přes arduino, když tam tenhle řádek není tak to hází: identifier "memset" is undefined

Reply:
Když přidáš `#include <cstring>`, tak to nefunguje?

Reply:
jo, funguje